src/helpers/buildWalletSections.tsx

```diff
@@ -386,6 +388,35 @@ const withBriefBalanceSection = (
     nativeCurrency
   );
 
+  const isEmpty = !briefAssets?.length && !collectibles?.length;
+  const isOnlyNFTs = !briefAssets?.length && collectibles?.length;
+
+  const emptySection = [
+    { type: 'BIG_EMPTY_WALLET_SPACER', uid: 'big-empty-wallet-spacer-1' },
```

Comment:
big asf

Reply:
should i rename to `CHONK_WALLET_SPACER`?